tests/cypress/integration/reportRepair/repairProblemBestDescription.spec.js

```diff
@@ -413,14 +457,7 @@ describe('repairProblemBestDescription', () => {
       cy.get('button').click();
     });
 
-    context('Walls, floor or ceiling, excluding damp', () => {
```

Comment:
it feels like we need these tests

Reply:
its still there if you scroll up, it looks like its been removed because of a merge conflict